deployment/terraform/batch.tf

```diff
@@ -0,0 +1,75 @@
+#
+# Batch resources
+#
+resource "aws_launch_template" "batch_gpu_container_instance" {
+  block_device_mappings {
+    device_name = "/dev/xvda"
+
+    ebs {
+      volume_size = 64
```

Comment:
I think that we should make this value a project level variable.

Looking at the manual CE that was created to bridge the gap until this task was completed, it doesn't have a launch template associated, so it seems unlikely that we'd pick a volume size smaller than the default.

Reply:
I made this value a project level variable and set it to the AMI's default of 30.